### DISCUSSION TITLE
[Q&A] Glpi 10.0.5 on Ubuntu 22 - SMTP mail not working

### DISCUSSION THREAD

**Discussion by Chico0008** [glpi-project/glpi-agent#321] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Glpi 10.0.0.5 hosted on a Ubuntu 22.04 serveur, with apache2/mariaDB

When configuring the Notification to mail / smtp, test mail to the admin imemdiately get an error notification.
tried again with debug mode, log from files/_log/mail-debug.log

```
2023-01-24 15:30:23 [500@s-web]
3 - Connection: opening to smtp.viry.net:8082, timeout=300, options=array()2023-01-24 15:30:23 [500@s-web]
3 - Connection: opened2023-01-24 15:30:23 [500@s-web]
2 - SERVER -> CLIENT: 2023-01-24 15:30:23 [500@s-web]
3 - Connection: closing due to error2023-01-24 15:30:23 [500@s-web]
3 - Connection: closed2023-01-24 15:30:23 [500@s-web]
3 - SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting2023-01-24 15:32:40 [500@s-web]
```

smtp.viry.net:8082 is a relay smtp server on windows server, and my ubuntu server (s-web.viry.net) au authorized to connect to the relay.

we have on old version en glpi (0.85) with the same notification smtp parameter working fine.
i'd like the new version work too.

**Comment by g-bougard** [comment]
Hi @Chico0008
please, this project is dedicated to glpi-agent. Use the glpi forum for such question.

**Reply by Chico0008** [reply]
ok, sorry
